mypy/messages.py

```diff
@@ -2510,8 +2531,11 @@ def match_statement_inexhaustive_match(self, typ: Type, context: Context) -> Non
     def iteration_dependent_errors(self, iter_errors: IterationDependentErrors) -> None:
         for error_info in iter_errors.yield_uselessness_error_infos():
             self.fail(*error_info[:2], code=error_info[2])
+        msu = mypy.typeops.make_simplified_union
```

Comment:
Please don't do this, the effect on readability is ~0, while it may potentially affect compiled performance.